read/parser.go

```diff
@@ -189,27 +188,13 @@ func (d *Dump) NumObjects() int {
 }
 func (d *Dump) Contents(i ObjId) []byte {
 	x := d.objects[i]
-	b := d.buf
-	if uint64(cap(b)) < x.Ft.Size {
-		b = make([]byte, x.Ft.Size)
-		d.buf = b
-	}
-	b = b[:x.Ft.Size]
-	n, err := d.r.ReadAt(b, x.offset)
-	if err != nil && !(n == len(b) && err == io.EOF) {
-		// TODO: propagate to caller
-		log.Fatal(err)
-	}
-	return b
+	return []byte(x.Contents)
```

Comment:
This does a copy of the contents for every call.  Probably you want to keep the Contents as a []byte.